inlang/source-code/server/src/main.ts

```diff
@@ -87,6 +89,17 @@ app.use(
 	})
 )
 
+app.use(
+	"/manage",
+	createProxyMiddleware({
+		target: manageAdress,
+		changeOrigin: true,
+		headers: {
+			Connection: "keep-alive",
+		},
+	})
+)
+
```

Comment:
if we want to host manage on `manage.inlang.com`, we don't need this proxy server

Reply:
@samuelstroschein but manage.localhost:3000 isn't working right? It's just for development, in production this isn't used.

Reply:
@flornkm correct. you need to open the correct localhost port directly. i don't find resources what better subdomain (manage.inlang.com) or subpath (inlang.com/manage). somewhat ambiguous decision here. 

**arguments for subdomain**

- better UX because users can type `manage*` and get autocomplete
- separation of the teams (no reliance on a common proxy server) 

**arguments for subpath**

- easier to deploy (?) no other domain handling etc

Reply:
Got it, I just thought it would be a better DX if we don't have to redirect to another local port.

Regarding the decision, let's go with subdomain. Additional arguments:

- subpaths on inlang.com are almost entirely SEO optimized, `inlang/manage` should be not indexed. So a subdomain is a better separation in my opinion.